snippets/variant-list-row.liquid

```diff
@@ -77,13 +80,75 @@
           {{ 'products.product.price.sale_price' | t }}
         </dt>
         <dd class="price">
-          <span>{{ 'sections.variant_list.each' | t: money: item_price }}</span>
+          {%- if product.quantity_price_breaks_configured? -%}
+            {%- liquid
+              assign volume_pricing_array = variant.quantity_price_breaks | sort: 'price'
+            -%}
+            <price-per-item
```

Comment:
Thoughts on making this a snippet?

Reply:
Eventually we will do it but maybe in a quieter time?